### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sajjaadf**
update Readme.me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
-chromebrew
+![](https://sajjaadf.github.io/chromebrew-Dev/images/brew.png) <br>
```
Comment (uberhacker) [inline README.md]: This url should be referencing this repo, not sajjaadf.github.io.

Comment (sajjaadf) [reply README.md]: Whoops

Comment (sajjaadf) [reply README.md]: Actually that is for the image

Comment (sajjaadf) [reply README.md]: Fine

Comment (sajjaadf) [reply README.md]: When the new website goes up as the pull request gets accepted, it will worj

**Comment by sajjaadf** [comment]
Depends on #2912

File: README.md (hunk 1 above)
Comment (uberhacker) [reply README.md]: Sure, it will work but the right way is to pull the image from this repo.

Comment (sajjaadf) [reply README.md]: fine

Comment (sajjaadf) [reply README.md]: fixed

**Pull request merged**